src/components/PersonalData/PersonalData.tsx

```diff
@@ -7,24 +7,10 @@ import {
 } from '@trussworks/react-uswds'
 import styles from './PersonalData.module.scss'
 import NavLink, { NavLinkProps } from 'components/util/NavLink/NavLink'
-import { useGetDisplayNameQuery } from 'operations/portal/queries/getDisplayName.g'
-import { useUser } from 'hooks/useUser'
 
-const PersonalData = () => {
-  const { user } = useUser()
+const PersonalData = ({ userDisplayName }: { userDisplayName: string }) => {
   const router = useRouter()
-
-  const { data } = useGetDisplayNameQuery()
-
-  let userDisplayName
-  if (data?.displayName) {
-    userDisplayName = data.displayName
-  } else if (user) {
-    userDisplayName = `${user.attributes.givenname} ${user.attributes.surname}`
-  }
```

Comment:
Praise: I like that this solution removes the need for doing this logic here and in `src/pages/api/graphql.tsx`